spec/models/group_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
```

Comment:
- [ ] Please kindly make sure all your tests are passing, some of them don’t pass 👎🏾 
- [ ] Also the App should have unit tests for all the most important components of the RoR application, kindly add more 🙃 

<img width="878" alt="Screenshot 2023-06-03 at 23 13 43" src="https://github.com/MarkoKermi/Budget_app/assets/74501523/84fdb762-de1d-465b-981a-6987754aca1a">